tests/system_tests/test_i22_system.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+from bluesky.run_engine import RunEngine
+
+from dodal.utils import make_all_devices
+
+if __name__ == "__main__":
+    """This test runs against the real beamline and confirms that all the devices connect
+    i.e. that we match the beamline PVs. Obviously this must be run on the DLS network
+    and could be flaky if parts of the beamline are down for maintenance.
+
+    This is not implemented as a normal pytest test as those tests run using the S03
+    EPICS ports and switching ports at runtime is non-trivial
```

Comment:
Nit: This isn't true when running against the real beamline

Reply:
What do you mean? It is true that it's not a normal pytest because switching at runtime is non-trivial?

Reply:
Sorry, should have specified, the ports point is not true. S03 runs on non-standard EPCIS port but a real beamline runs on the standard ones.